### ISSUE TITLE
[BUG] PHP message: PHP Fatal error:  Uncaught Error: Call to undefined function get_magic_quotes_gpc()

### ISSUE THREAD

**Issue by Raistlfiren** [linuxserver/docker-piwigo#66]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

After a fresh install, I am receiving the following error `PHP message: PHP Fatal error:  Uncaught Error: Call to undefined function get_magic_quotes_gpc()` in the NGINX PHP FPM logs.

### Steps To Reproduce

1. Install the app from Docker
2. Load the page - blank page
3. View the logs

### Environment

```markdown
- OS: Unraid
- How docker service was installed: Unraid App Center
```

### CPU architecture

x86-64

### Docker creation

```bash
Default run command.
```

### Container logs

```bash
[migrations] started
[migrations] 01-nginx-site-confs-default: skipped
[migrations] done
───────────────────────────────────────

      ██╗     ███████╗██╗ ██████╗ 
      ██║     ██╔════╝██║██╔═══██╗
      ██║     ███████╗██║██║   ██║
      ██║     ╚════██║██║██║   ██║
      ███████╗███████║██║╚██████╔╝
      ╚══════╝╚══════╝╚═╝ ╚═════╝ 

   Brought to you by linuxserver.io
───────────────────────────────────────

To support LSIO projects visit:
https://www.linuxserver.io/donate/

───────────────────────────────────────
GID/UID
───────────────────────────────────────

User UID:    99
User GID:    100
───────────────────────────────────────

using keys found in /config/keys
**** The following active confs have different version dates than the samples that are shipped. ****
**** This may be due to user customization or an update to the samples. ****
**** You should compare the following files to the samples in the same folder and update them. ****
**** Use the link at the top of the file to view the changelog. ****
┌────────────┬────────────┬────────────────────────────────────────────────────────────────────────┐
│  old date  │  new date  │ path                                                                   │
├────────────┼────────────┼────────────────────────────────────────────────────────────────────────┤
│            │ 2023-04-13 │ /config/nginx/nginx.conf                                               │
│            │ 2023-04-13 │ /config/nginx/site-confs/default.conf                                  │
└────────────┴────────────┴────────────────────────────────────────────────────────────────────────┘
[custom-init] No custom files found, skipping...
[ls.io-init] done.
```

**Comment by lmoellendorf** [comment]
I am a user myself and I have a similar issue. After months I restarted my piwigo docker and I only get a blank page. 
@Raistlfiren, can you please tell me in which log you found the PHP error so I can confirm that I get the same?

**Comment by doodhout** [comment]
The error can be found in `/config/log/nginx/error.log`.

I have the same problem here.

**Comment by doodhout** [comment]
Fixed! I reset my container for Piwigo 11.5 using a new container for Piwigo 13.8 which comes bundled with PHP 8, but Piwigo was only made PHP 8 compatible starting from version 12.

The correct way to upgrade: first in the admin section, perform the in-app upgrade to the latest version before resetting the container using a newer image, because Piwigo specific files will remain despite the container refresh.

This was the complete error message I got: 

```
2023/09/06 15:59:42 [error] 281#281: *10 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught Error: Call to undefined function get_magic_quotes_gpc() in /config/www/gallery/include/common.inc.php:20
Stack trace:
#0 /config/www/gallery/index.php(11): include_once()
#1 {main}
  thrown in /config/www/gallery/include/common.inc.php on line 20" while reading response header from upstream, client: 172.17.0.1, server: _, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9000", host: "192.168.1.2:8080"
```
